### DISCUSSION TITLE
Multiline-entry for customtkinter

### DISCUSSION THREAD

**Discussion by StandardV** [TomSchimansky/CustomTkinter#1199] [Q&A] [answered]
Exactly like the title, as of right now, is there anyway that we can use a multiline-entry widget while still preserve the entry format when pasting with "\n" in customtkinter? Thank you!

**Comment by Wolf-SO** [comment, marked as answer]
What about [CTkTextbox](https://github.com/TomSchimansky/CustomTkinter/wiki/CTkTextbox)?

**Reply by StandardV** [reply]
It seem that I didn't notice there's a widget like this, my apologies! thank you for you reply
